src/index.html

```diff
@@ -1,13 +1,65 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body class="page__content">
+    <div class="card page__card" data-qa="card">
+      <img
+        class="card__product-image card__product-image--wrapper"
+        src="images/imac.jpeg"
+        alt="Apple iMac 27inch image"
+      >
+
+      <div class="card__product-info card__product-info--wrapper">
+        <span class="card__product-name card__product-name--wrapper">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </span>
+
+        <span class="card__product-code card__product-code--wrapper">
+          Product code: 195434
+        </span>
+
+        <div class="card__rating card__rating--wrapper">
+          <div class="stars stars--4">
+            <div class="stars__star stars__star--wrapper"></div>
```

Comment:
why do u need second class & it's definately not wrapper

Reply:
Second class is needed to make gaps between stars, because I want to follow the BEM methodology.
Hmm, I think so. Perhaps the name like "stars__star--right-gap_4px" is more suitable in this case

Reply:
stars__star isn't block, it is element so u can set gaps there